locale/en_US/LC_MESSAGES/clr-installer.po

```diff
@@ -239,8 +239,8 @@ msgstr "RESCAN MEDIA"
 msgid "Adding extra users"
 msgstr "Adding extra users"
 
-msgid "Username must contain only numbers, letters, commas, - or _"
-msgstr "Username must contain only numbers, letters, commas, - or _"
+msgid "Username must contain only numbers, letters, commas, - or _ "
```

Comment:
There is whitespace at the end of this string.

Reply:
Actually My editor kept complaining that a string cant end with a . or _. So i added this

Reply:
But the editor is wrong.  😃

Reply:
Done